en/docs/apis/api-concepts.md

```diff
@@ -1,23 +1,69 @@
 # API Concepts
-Before you start managing APIs via Choreo, there are basic concepts that you need to understand. This section walks you through all the concepts you need to know. 
+Choreo's API management capabilities allow you to create APIs, manage APIs, and expose services and external APIs as managed APIs. Here are some basic concepts which will help you understand API management in Choreo as you create and manage your APIs in the [Choreo Console]({{choreo_console}}).
  
 ## API
-An API in Choreo allows you to expose business functionality as a service to customers, partners, suppliers, and other stakeholders.
+An API is an intermediate layer that acts as a communication protocol between a consumer and a service, simplifying the consumption of the service. In addition to hiding the underlying implementation details of a service, an API provides a secure, controlled, and well-documented approach to access an exposed service.
 
-## API Management Components
-Described below are the key components of Choreo API management:
+## API format
+OpenAPI Specification (OAS, a.k.a Swagger) format is the underlying representation of an API in Choreo. You can import an API using a Swagger definition in the Choreo console. Additionally, a user can view, edit, import, or download an API definition in OAS format via the Choreo console.
```

Comment:
Shouldn't we mention that users can create an API  from scratch too?

Reply:
You can't actually create an API from scratch in swagger. You have to create an API, and then view the source, right?

Reply:
In choreo we allow creating from scratch as well. This is high level API concepts page. So why we are talking only about creating from OpenAPI stuff here. We need to reword mentioning the other option as well. 
<img width="1606" alt="image" src="https://user-images.githubusercontent.com/16300038/120242966-e72e2f00-c283-11eb-9ac9-b65f73512f45.png">

Reply:
You have to design the API from scratch via the UI. It doesn't open you the swagger editor to edit the API. Here we are talking about what you can do with the API definition.  And we can't create an API from scratch through the editor, by writing the API definition. The underlying format is the OpenAPI format.

Reply:
Yes. Was thinking since this is highlevel API concepts page, shouldn't we at least mention about some concepts related to the ability to start from scratch (not under this sub-topic) . But if we don't have anything to say specifically, lets go with the current one.

Reply:
We are mentioning it in the first sentence in the document. "Choreo's API management capabilities allow you to create APIs, manage APIs, and expose services and external APIs as managed APIs. " So I guess we are good to go.